### PR TITLE
Improve Readme (#18)

### PR DESCRIPTION
**Pull request by kanishk1010**
I have changed the readme but it still needs some work. Would collaborate with @snazzybunny to finalize it.

Here's the [preview](https://github.com/kanishk1010/treehouse-builder/blob/readme/README.md)

### CODE REVIEW

**Comment by kanishk1010** [comment]
Any suggestions for the 'Built with' section? like maybe docker etc.??

**Comment by snazzybunny** [comment]
@kanishk1010 @dogi We will need to update the README.md to reflect the autorun changes because there are no more CUSTOM_COMMANDS etc.

**Comment by Ebuka-Chibueze** [comment]
@kanishk1010 @dogi @snazzybunny Do you think a similar PR should be made for [ole--vagrant-treehouses](https://github.com/ole-vi/ole--vagrant-treehouses)?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,68 @@
-Start with:
-`./treehouse-builder --chroot`
+# treehouse-builder
+
+Treehouse-builder is based on [Raspian](https://www.raspbian.org/) and allows the user to develop and tailor their own custom RPi images. The script will modify the latest Raspian image by installing packages, purging packages and executing custom commands, and then finally creates a bootable .img file that can be burned to the microSD card.
```
Comment (mappuji) [inline README.md]: Rasbian instead of Raspian isn't it?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,68 @@
...
+
+## Instructions
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
```
Comment (mappuji) [inline README.md]: Operating system requirement?

File: README.md [hunk 3]
```diff
@@ -1,10 +1,68 @@
...
+System requirements
+class10 microSD card (minimum 8 Gb)
+
+Required packages
+kpartx wget gpg parted qemu-arm-static
+
+To install the required packages, run the following command in Debian/Ubuntu: 
+sudo apt-get install kpartx wget gpg parted qemu-arm-static. 
+```
+### Getting Started
```
Comment (mappuji) [inline README.md]: How about the different between running `treehouse-builder` in vagrant vs in host OS?

Comment (kanishk1010) [inline README.md]: @mappuji Thank you for pointing out the corrections and improvements. I have made changes accordingly. For this one, I have added a link to ole--vagrant-treehouses, which will have instructions for the same.

**Pull request merged**